numpy/__init__.pyi

```diff
@@ -408,7 +408,10 @@ dstack: Any
 ediff1d: Any
 expand_dims: Any
 extract: Any
-eye: Any
+eye: Callable[
```

Comment:
Merely annotating the function as `Callable` won't be enough to capture its signature unfortunately, as the former is unable to describe keyword arguments.

See `np.array` for a potential example: https://github.com/numpy/numpy/blob/eb4317d0a4c94b3f1454dca58c76caed8bbdc7e6/numpy/__init__.pyi#L2621-L2630